src/dmd/doc.d

```diff
@@ -1948,6 +2007,201 @@ Lno:
     return i;
 }
 
+/****************************************************
+ * Replace the inline link portion (the part in parentheses) of a Markdown link.
+ */
+size_t replaceMarkdownInlineLink(OutBuffer* buf, size_t i, MarkdownDelimiter delimiter)
+{
+    if (buf.data[i+1] == '(')
```

Comment:
I suggest negating the condition and do an early return to save some indentation for the test of the code.